README.md

```diff
@@ -484,7 +484,7 @@ Put this file in `/bin`:
 * `systemd-email`: Sends email using sendmail(1). This script also features time-out for not spamming Gmail servers and getting my account blocked.
 
 Put this file in `/etc/systemd/system/`:
-* `status-email-user@.service`: A service that can notify you via email when a systemd service fails. Edit the target email address in this file.
+* `status-email-user@.service`: A service that can notify you via email when a systemd service fails. Edit the target email address in this file, and replace or remove `{{ INSTALL_PREFIX }}` according to your installation.
```

Comment:
Installing with the Makefole should not require to manually edit this file, right?

Reply:
Nope, as @giuaig pointed out in the other talk, this is an optional feature and intended to be manually installed according to the README. It is totally ignored by `make` 😇 right now.

I'll address your other comments later!

Reply:
Ah gotcha!